File: fixtures/hub-cobol-clbs-mini/batch/reference_emit_cardintrn.py
"""Reference emit for CARDINTRN — CardDemo fee idiom 1000 * 0.029 → 29.00."""
# EXPECTED: 29.00
BAL = 1000.00
RATE = 0.0290
FEE = round(BAL * RATE + 1e-12, 2)
print(f"{FEE:.2f}")

File: fixtures/hub-cobol-clbs-mini/batch/reference_emit_cardintrn.java
// EXPECTED: 29.00
// Reference emit contract for CARDINTRN (1000.00 * 0.0290 rounded).
public final class CardIntrnRef {
  public static void main(String[] args) {
    double bal = 1000.00;
    double rate = 0.0290;
    double fee = Math.round(bal * rate * 100.0) / 100.0;
    System.out.printf("%.2f%n", fee);
  }
}
